integration/feature/keyword-module/resources/import/package.mill

```diff
@@ -0,0 +1,7 @@
+package build.`import`
+import mill._, scalalib._
+
+object `package` extends RootModule with ScalaModule {
```

Comment:
Let's not make this a `ScalaModule`, since we don't care about the scala side at all. Instead, we can just add a trivial `def  task = Task { 123 }`

Furthermore, let's exercise a few more Scala keywords, with `this/package.mill`, `for/package.mill`, `if/package.mill`, just to make sure it works for more than just the single `import` keyword

Reply:
I added few more case. One interesting point is that `` `package` `` will fail but I assume it's not due to backtick as mill complain this way:
```
[2758] [build.mill-57] [error] /home/Chikei/repos/mill/out/integration/feature/keyword-module/local/test.dest/sandbox/run-1/out/mill-build/generateScriptSources.dest/build_/build.mill:21:21: lazy value millDiscover overrides nothing
[2758] [build.mill-57] [error]   override lazy val millDiscover: _root_.mill.define.Discover = _root_.mill.define.Discover[this.type]
[2758] [build.mill-57] [error]                     ^
[2758] [build.mill-57] [error] one error found
```
So I just removed it from cases for now.